make/jdk/src/classes/build/tools/generatecharacter/GenerateCharacter.java

```diff
@@ -1190,7 +1190,7 @@ static void genTable(StringBuffer result, String name,
         if (Csyntax)
             result.append("  static ");
         else
-            result.append("  static final ");
+            result.append("  @Stable static final ");
         result.append(atype);
         result.append(" ").append(name).append("[");
```

Comment:
While you are at it - when Csyntax == false, can we make it print `atype[] name` instead of `atype name[]`?

Reply:
This will require more changes, and has no benefits other than better coding style.

Reply:
Reasonable. These generated classes have a lot of code improvement opportunities that do not affect program semantics. We should handle those in a dedicated RFE.